### ISSUE TITLE
"Open Babun here" in Explorer doesn't switch to directory selected

### ISSUE THREAD

**Issue by gliptak** [babun/babun#390]
Looking at the registry entry HKEY_CLASSES_ROOT\Directory\shell\babun\command:
```
C:\.babun\cygwin\bin\mintty.exe /bin/env CHERE_INVOKING=1 /bin/zsh.exe
```
So I'm not sure how the current location is passed in from Explorer.

Changing this above to:
```
C:\.babun\cygwin\bin\mintty.exe /bin/env CHERE_INVOKING=1 /bin/zsh.exe -c \"cd \\\"`cygpath -u '%L'`\\\"
```
Corrected this. Please comment. Thanks

**Comment by kubamarchwicki** [comment]
Can you elaborate a bit on your test scenario? According to cygwin documentation that should be sufficient and it works fine in the test I was deriving. 
What operating system? You click on driver icon, folder icon, within folder?

**Comment by gliptak** [comment]
Certainly. I tried some combinations (on Win7). It works from the Details Pane (right) for local/mounted/UNC drives. Doesn't work from Navigation Pane (left tree view) for local/mounted/UNC drives. Works for local Desktop items, doesn't work for mounted/UNC dekstop items. (I also might have observed windows already being open having some effect, but no longer believe that being the case)
I also have (msysgit) Git Bash installed, which seems to work for all scenarios (above).

**Comment by FichteFoll** [comment]
The "current location" is inferred from the current working directory.

**Comment by gliptak** [comment]
I just retried this. Still doesn't work from Navigation Pane (left tree view) for local/mounted/UNC drives.

**Comment by flyisland** [comment]
same issue on Windows 7 with `babun version 1.2.0`

>  It works from the Details Pane (right) for local/mounted/UNC drives. Doesn't work from Navigation Pane (left tree view) for local/mounted/UNC drives.

**Comment by nainwal** [comment]
i can't go to desktop on Babun when i used $ cd ~/Desktop......
its show sthe error:-zsh: command not found: $
